addons/nametags/functions/fnc_onDraw.sqf

```diff
@@ -9,6 +9,9 @@ if ((ctrlPosition _ctrl) isNotEqualTo _pos) then {
 };
 
 private _target = cursorObject;
+private _skipVicCheck = (netId _target) isEqualTo "1:0"; // only ever true when cursorObject returns the weapon of a
+// unit, per the comment from Pierre MGI at https://community.bistudio.com/wiki/cursorObject
+// we'd like to find the cursorObject regardless, so we piggyback on the LIS check to find the most probable one
```

Comment:
Move that into the if execution block of `GVAR(useLIS)` as it is only needed in there.